src/osp/Resource/AssetImporter.cpp

```diff
@@ -67,20 +67,23 @@ void osp::AssetImporter::load_sturdy_file(std::string const& filepath, Package&
         std::cout << "Error: couldn't open file: " << filepath << "\n";
     }
 
-    load_sturdy(gltfImporter, pkg);
+    // repurpose filepath into unique identifier for file resources
```

Comment:
filePath looks an aweful lot like it should be an std::string_view.

Reply:
It probably should, but since I didn't introduce it in this PR, I think it would be better to make one giant PR where we go and change all the strings to string_views at once, rather than gradually changing them in unrelated PRs

Reply:
Sure that's fine.